README.md

```diff
@@ -3,3 +3,17 @@
 [![Gitter](https://badges.gitter.im/eclipse/microprofile-starter.svg)](https://gitter.im/eclipse/microprofile-starter?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 [MicroProfile starter - Generate MicroProfile Maven Project](https://test-start.microprofile.io/index.xhtml)
+
+The MicroProfile Starter is a community-driven project that provides guided examples for the different APIs under the MicroProfile specification. The examples, based on best practices, are code samples to help you get familiar with MicroProfile APIs so that you can get started quickly on developing and implementing your business logic.
```

Comment:
I would not call this `based on best practices`. We just try to give some examples within the limited possibilities of a small example application.